### ISSUE TITLE
Improve accessibility for usage with the TalkBack screen-reader

### ISSUE THREAD

**Issue by schulle4u** [FredJul/Flym#422]
Flym already is quite accessible for usage with the TalkBack screen-reader in most parts. There however are some buttons which should be labeled to improve accessibility. Currently the following buttons only provide an element ID to the screen-reader or are just unrecognized: 

* The feed menu button in the upper left side
* The mark all as read button
* The "more" and "expand" buttons for groups in the feed menu

TalkBack is able to read out element IDs in the latest version which makes many apps more accessible, but this function is not activated by default. Therefore propper labeling is always the better solution. 

Furthermore the feed reordering function doesn't seem to be accessible for TalkBack users. At least I was not able to find out how to assign a feed to a group or move feeds between groups. How is this done visually? Maybe there is a gesture which talkback isn't able to reproduce. 

Steffen

**Comment by Echolon** [comment]
Hi,

even I see this is important in terms of accessibility, I think with the current ressources its out of scope. Maybe there are other willing projects out there.
However, PR is welcome

**Comment by FredJul** [comment]
Thanks to amkcpu, this area should be improved. I'll soon publish a new version of the app, you will be able to test soon if this is really better or not.
